go/sqltypes/value.go

```diff
@@ -318,6 +318,17 @@ func (v Value) ToInt() (int, error) {
 	return strconv.Atoi(v.RawStr())
 }
 
+// ToFloat32 first parses the value as MySQL would return it as a float64, then
+// downcasts it to a float32. It may overflow if the value exceeds 32 bits.
+func (v Value) ToFloat32() (float32, error) {
```

Comment:
it would be a bit clunkier for me to do this inline in `grpcvtctldserver`, but this is also very much not a safe cast to do in general so i'm happy to move it there and grumble about the clunkiness :)

Reply:
actually, using `float32(.AsFloat64())` works just as well for me